.buildkite/pipelines/pull_request/security_solution.yml

```diff
@@ -12,3 +12,14 @@ steps:
           limit: 1
     artifact_paths:
       - "target/kibana-security-solution/**/*"
+
+  - command: .buildkite/scripts/steps/functional/security_solution_burn.sh
+    label: ':fire: Security Solution Cypress Tests'
```

Comment:
Nit: to make it a bit more explicit about what it does, could name the job as `Security Solution Cypress tests, burning changed specs`